src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -973,6 +982,26 @@ public void run() {
         }
     }
 
+    /**
+     * Verifies the status messages in the page are same as the messages specified.
```

Comment:
`Verifies the texts for status messages to the user in the page are equal to the expected texts.`

Reply:
okay